1-exercises/H-array-methods-2/exercise.js

```diff
@@ -15,8 +15,8 @@ let everyone = [
   "Swathi",
 ];
 
-let firstFive; // complete this statement
-let lastFive; // complete this statement
+let firstFive = everyone.slice(0, 5); // complete this statement
+let lastFive = everyone.slice(everyone.length - 5, everyone.length); // complete this statement
```

Comment:
There's an easier way of doing this 
```suggestion
let lastFive = everyone.slice(-5)
```